REST_API_DOCS.md

```
@@ -0,0 +1,1216 @@
+}
+```
+
+---
+
+| 파라미터 | 타입 | 필수 | 설명 |
+| --- | --- | --- | --- |
+| sort | string | X | 정렬 기준 (latest, views, likes) |
+| page | integer | X | 페이지 번호 (기본값: 1) |
+| limit | integer | X | 페이지당 게시글 수 (기본값: 20) |
+
+**Path Parameters**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)